tests/functional/export_v2/cram/minify-output.t

```diff
@@ -0,0 +1,31 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Tree with a few nodes is not automatically minified.
+
+  $ ${AUGUR} export v2 \
+  >   --tree "$TESTDIR/../data/tree.nwk" \
+  >   --output output.json &>/dev/null
+
+  $ head -c 20 output.json
+  {
+    "version": "v2", (no-eol)
+
+It can be forcefully minified with an argument.
+
+  $ ${AUGUR} export v2 \
+  >   --tree "$TESTDIR/../data/tree.nwk" \
+  >   --minify-json \
+  >   --output output.json &>/dev/null
+
+  $ head -c 20 output.json
+  {"version": "v2", "m (no-eol)
+
+Tree with many nodes is automatically minified.
+
+FIXME: find a tree with >1000 nodes for testing here.
```

Comment:
Not sure where to find an appropriate file to add for this test.

Reply:
Why not make one dynamically? It can just be a 1000 node polytomy. After spending a bunch of time writing augur tests recently my conclusion is that committing big datasets (although still small file sizes) is not the best way to go.

Reply:
Good point, I'll look into doing that. I also prefer creating test files within the Cram file - it's easier to see what's being tested.

Reply:
Done in [force-push](https://github.com/nextstrain/augur/compare/fb82a794b9036b46c3e7d5be82678d06ed4c7927..5e43b3de331e93ef7ad6ca86ff2ff655e8d869e5).

Reply:
Generating the 1000-node tree adds ~5s to test run time, which I think is fine.